tests/dash/dash_acl.py

```diff
@@ -155,7 +155,7 @@ def config(self):
             ACL_PRIORITY: 255,
             ACL_ACTION: self.default_action,
             ACL_TERMINATING: "false",
-            ACL_PROTOCOL: "17",
+            ACL_PROTOCOL: "17, 6, 1, 2",
```

Comment:
why IGMP required?

Reply:
You are right. We don't need it. I have removed it.
Thanks. Please help review it again.